### PR TITLE
[ready to publish] Spanish translation

### PR DESCRIPTION
**Pull request by rumoroso**

### CODE REVIEW

**Comment by shawna-slh** [comment]
Hi Jorge,

In the footer text (which is near the top of the source page), please do *translate* "Updated 10 May 2019."

(I will update the instructions so it's more clear to translate the date, yet not change the date. :-)

For the list items linked under "WCAG", please check that they are the same as in [WCAG 2 Spanish Translation](http://www.sidar.org/traducciones/wcag20/es/). If you have any questions about the text, you can work with Martin/@espinr and Emmanuelle on wording.

Thanks!

**Comment by rumoroso** [comment]
@espinr , could you please collaborate in the review or let me know who could do it?
Thanks in advance

**Comment by rumoroso** [comment]
Hi @slhenry 

> In the footer text (which is near the top of the source page), please do _translate_ "Updated 10 May 2019."

Sure

> For the list items linked under "WCAG", please check that they are the same as in [WCAG 2 Spanish Translation](http://www.sidar.org/traducciones/wcag20/es/).> For the list items linked under "WCAG", please check that they are the same as in [WCAG 2 Spanish Translation](http://www.sidar.org/traducciones/wcag20/es/).

If I understood correctly, you mean that the links should point to the corresponding sections in that document.

I'll update everything and commit the changes.

Thank you for your really fast feed back :)

**Comment by espinr** [comment]
> @espinr , could you please collaborate in the review or let me know who could do it?

Sure!

File: index.es.md [hunk 1]
```diff
@@ -0,0 +1,720 @@
...
+{% include excol.html type="start" %}
+
+#### Historias de usuario relacionadas con alternativas de texto
+{:#stories-related-to-text-alternatives.no_toc}
+
+{% include excol.html type="middle" %}
+
+-   [Alex, reportero con lesión por esfuerzo repetitivo](/people-use-web/user-stories/#reporter)
+-   [Martine, estudiante online con problemas de audición](/people-use-web/user-stories/#onlinestudent)
+-   [Ilya, ejecutiva senior que con ceguera](/people-use-web/user-stories/#accountant)
```
Comment (espinr) [inline index.es.md]: minor typo: 
`[Ilya, ejecutiva senior con ceguera](/people-use-web/user-stories/#accountant)`

File: index.es.md [hunk 2]
```diff
@@ -0,0 +1,720 @@
...
+-   [Preety, estudiante de secundaria con trastorno por déficit de atención e hiperactividad y dislexia](/people-use-web/user-stories/#classroomstudent)
+-   [Yun, retirada, con baja visión, temblores en manos y leve perdida de memoria a corto plazo](/people-use-web/user-stories/#retiree)
```
Comment (espinr) [inline index.es.md]: proposal: 
baja visión > visión reducida

File: index.es.md [hunk 3]
```diff
@@ -0,0 +1,720 @@
...
+-   [Kaseem, adolescente con sordera y ceguera](/people-use-web/user-stories/#teenager)
+
+{% include excol.html type="end" %}
+
+### Subtítulos y otras alternativas para multimedia {#captions}
+
+Las personas que no pueden oir audio o ver video necesitan alternativas. Entre los ejemplos se incluyen:
```
Comment (espinr) [inline index.es.md]: oir > oír

File: index.es.md [hunk 4]
```diff
@@ -0,0 +1,720 @@
...
+{% include excol.html type="end" %}
+{% include excol.html type="start" %}
+
+#### Historias de usuario relacionadas con multimedia {#stories-related-to-multimedia}
+{:.no_toc}
+
+{% include excol.html type="middle" %}
+
+-   [Martine, estudiante online con problemas de audición](/people-use-web/user-stories/#onlinestudent)
+-   [Ilya, ejecutiva senior que con ceguera](/people-use-web/user-stories/#accountant)
```
Comment (espinr) [inline index.es.md]: minor typo:
`[Ilya, ejecutiva senior con ceguera](/people-use-web/user-stories/#accountant)`

File: index.es.md [hunk 5]
```diff
@@ -0,0 +1,720 @@
...
+-   [Kaseem, adolescente con sordera y ceguera](/people-use-web/user-stories/#teenager)
+
+{% include excol.html type="end" %}
+
+### Contenido que puede ser presentado de diferentes maneras {#adaptable}
+
+Para que los usuarios sean capaces de cambiar la presentación del contenido, es necesario que:
+
+-   Encabezados, listas, tablas, campos de formulario y estructuras de contenido utilicen el marcado apropiado
+-   El orden de la información o la secuendia de instrucciones sean independientes de la presentación
```
Comment (espinr) [inline index.es.md]: secuendia > secuencia

File: index.es.md [hunk 6]
```diff
@@ -0,0 +1,720 @@
...
+-   [Apartado B. Soporte la producción de contenido accesible](https://www.w3.org/TR/ATAG20/#part_b)
+
+{% include excol.html type="end" %}{% include excol.html type="start" %}
+
+#### Historias de usuario relacionadas con adaptabilidad
+{:.no_toc}
+
+{% include excol.html type="middle" %}
+
+-   [Lee, comprador online con ceguera para el color](/people-use-web/user-stories/#shopper)
```
Comment (espinr) [inline index.es.md]: proposal:
`[Lee, comprador online con daltonismo](/people-use-web/user-stories/#shopper)`

File: index.es.md [hunk 7]
```diff
@@ -0,0 +1,720 @@
...
+-   [Alex, reportero con lesión por esfuerzo repetitivo](/people-use-web/user-stories/#reporter)
+-   [Ilya, ejecutiva senior que con ceguera](/people-use-web/user-stories/#accountant)
+-   [Preety, estudiante de secundaria con trastorno por déficit de atención e hiperactividad y dislexia](/people-use-web/user-stories/#classroomstudent)
+-   [Yun, retirada, con baja visión, temblores en manos y leve perdida de memoria a corto plazo](/people-use-web/user-stories/#retiree)
```
Comment (espinr) [inline index.es.md]: proposal:
baja visión > visión reducida

File: index.es.md [hunk 8]
```diff
@@ -0,0 +1,720 @@
...
+-   [Apartado B. Soporte la producción de contenido accesible](https://www.w3.org/TR/ATAG20/#part_b)
+
+{% include excol.html type="end" %}{% include excol.html type="start" %}
+
+#### Historias de usuario relacionadas con peceptibilidad
+{:.no_toc}
+
+{% include excol.html type="middle" %}
+
+-   [Lee, comprador online con ceguera para el color](/people-use-web/user-stories/#s#reporter)
```
Comment (espinr) [inline index.es.md]: proposal: daltonismo

File: index.es.md [hunk 9]
```diff
@@ -0,0 +1,720 @@
...
+-   [Martine, estudiante online con problemas de audición](/people-use-web/user-stories/#reporter)
+-   [Ilya, ejecutiva senior que con ceguera](/people-use-web/user-stories/#accountant)
```
Comment (espinr) [inline index.es.md]: `[Ilya, ejecutiva senior con ceguera](/people-use-web/user-stories/#accountant)`

File: index.es.md [hunk 10]
```diff
@@ -0,0 +1,720 @@
...
+-   [Yun, retirada, con baja visión, temblores en manos y leve perdida de memoria a corto plazo](/people-use-web/user-stories/#retiree)
+-   [Kaseem, adolescente con sordera y ceguera](/people-use-web/user-stories/#teenager)
+
+{% include excol.html type="end" %}
+
+## Interfaz de usuario y navegación operables {#operable}
+
+### Funcionalidad está disponible desde el teclado {#keyboard}
```
Comment (espinr) [inline index.es.md]: proposal:
`### Funcionalidad disponible desde el teclado {#keyboard}`

File: index.es.md [hunk 11]
```diff
@@ -0,0 +1,720 @@
...
+
+Muchas personas no utilizan el ratón y dependen del teclado para interactuar con la Web. Esto requiere acceso mediante teclado a todas las funciones, incluidos los controles de formulario, entrada de datos y otros componentes de la interfaz de usuario.
+
+La accesibilidad con teclado incluye:
+
+-   Toda la funcionalidad que está disponible con ratón también está disponible con teclado
+-   El foco del teclado no queda atrapado en ninguna parte del contenido
+-   Los navegadores web, herramientas de autor y otras herramientas facilitan soporte para teclado
+
+El cumplimiento de este requisito ayuda a los usuarios de teclado, incluidos quienes que utilizan teclados alternativos, como teclados con diseños ergonómicos, teclados en pantalla o dispositivos de conmutación. También ayuda a las personas que utilizan reconocimiento de voz (entrada de voz) para operar sitios web y dictar texto a través de la interfaz del teclado.
```
Comment (espinr) [inline index.es.md]: (entrada de voz) > (entrada por voz)?

File: index.es.md [hunk 12]
```diff
@@ -0,0 +1,720 @@
...
+
+{% include excol.html type="end" %}{% include excol.html type="start" %}
+
+#### Historias de usuario relacionadas con la accesibilidad de teclado
+{:.no_toc}
+
+{% include excol.html type="middle" %}
+
+-   [Alex, reportero con lesión por esfuerzo repetitivo](/people-use-web/user-stories/#reporter)
+-   [Ilya, ejecutiva senior que con ceguera](/people-use-web/user-stories/#accountant)
```
Comment (espinr) [inline index.es.md]: `[Ilya, ejecutiva senior con ceguera](/people-use-web/user-stories/#accountant)`

**Comment by shawna-slh** [comment]
> If I understood correctly, you mean that the links should point to the corresponding sections in that document.

Oh dear, I was not clear enough. (sad face)
I just meant that the *text* wording of the link should be the same *text* as in the translation.

Although, it makes sense to link to the Spanish translation once it is complete.
_[Update: Except the English version of "Accessibility Principles" links to the Quick Reference, which has more information than the WCAG 2 document itself...]_

Please leave it as you have it for now and I will figure out what to do...

**Comment by rumoroso** [comment]
Hi, it was not your fault, but just my misunderstanding and that it is the first time I do this collaboration, so just time and experience for being aligned in what we say and others understand ;)

> Oh dear, I was not clear enough. (sad face)
> I just meant that the _text_ wording of the link should be the same _text_ as in the translation.

**Comment by shawna-slh** [comment]
I greatly appreciate your contributions and collaboration.

We are refining our translations instructions and process, too. So I also appreciate your patience. :-)

File: index.es.md [hunk 13]
```diff
@@ -0,0 +1,720 @@
...
+
+{% include excol.html type="end" %}
+
+### Los usuarios tienen tiempo suficiente para leer y utilizar el contenido {#time}
+
+Algunas personas necesitan más tiempo que otras para leer y utilizar el contenido. Por ejemplo, hay quienes requieren más tiempo para escribir texto, comprender instrucciones, operar controles o incluso para completar tareas en un sitio web.
+
+Entre los ejemplos de como proporcionar tiempo suficiente se incluyen los mecanismos para:
```
Comment (espinr) [inline index.es.md]: como > cómo

File: index.es.md [hunk 14]
```diff
@@ -0,0 +1,720 @@
...
+
+-   Detener, extender o ajustar los límites de tiempo, excepto cuando sea necesario
+-   Pausar, detener u ocultar movimiento, parpadeo o scroll de contenido
```
Comment (espinr) [inline index.es.md]: proposal: scroll > desplazamiento del

File: index.es.md [hunk 15]
```diff
@@ -0,0 +1,720 @@
...
+-   Posponer o suprimir interrupciones, excepto cuando sean necesarias
+-   Re-autenticar cuando la sesion expire sin perdida de datos
```
Comment (espinr) [inline index.es.md]: sesión

File: index.es.md [hunk 16]
```diff
@@ -0,0 +1,720 @@
...
+{% include excol.html type="end" %}{% include excol.html type="start" %}
+
+#### Historias de usuario relacionadas con tiempo suficiente
+{:.no_toc}
+
+{% include excol.html type="middle" %}
+
+-   [Alex, reportero con lesión por esfuerzo repetitivo](/people-use-web/user-stories/#reporter)
+-   [Preety, estudiante de secundaria con trastorno por déficit de atención e hiperactividad y dislexia](/people-use-web/user-stories/#classroomstudent)
+-   [Yun, retirada, con baja visión, temblores en manos y leve perdida de memoria a corto plazo](/people-use-web/user-stories/#retiree)
```
Comment (espinr) [inline index.es.md]: proposal: visión reducida

File: index.es.md [hunk 17]
```diff
@@ -0,0 +1,720 @@
...
+-   [Luis, asistente en un supermercado con síndrome de Down](/people-use-web/user-stories/#supermarketassistant)
+-   [Kaseem, adolescente con sordera y cegera](/people-use-web/user-stories/#teenager)
```
Comment (espinr) [inline index.es.md]: cegera > ceguera

File: index.es.md [hunk 18]
```diff
@@ -0,0 +1,720 @@
...
+-   [Apartado B. Soporte la producción de contenido accesible](https://www.w3.org/TR/ATAG20/#part_b)
+
+{% include excol.html type="end" %}{% include excol.html type="start" %}
+
+#### Historias de usuario relacionadas con compatibilidad
+{:.no_toc}
+
+{% include excol.html type="middle" %}
+
+-   [Lee, comprador online con cegera para el color](/people-use-web/user-stories/#shopper)
```
Comment (espinr) [inline index.es.md]: cegera > ceguera

File: index.es.md [hunk 19]
```diff
@@ -0,0 +1,720 @@
...
+-   [Alex, reportero con lesión por esfuerzo repetitivo](/people-use-web/user-stories/#reporter)
+-   [Ilya, profesional senior con cegera](/people-use-web/user-stories/#accountant)
+-   [Preety, estudiante de secundaria con trastorno por déficit de atención e hiperactividad y dislexia](/people-use-web/user-stories/#classroomstudent)
+-   [Luis, asistente en un supermercado con síndrome de Down](/people-use-web/user-stories/#supermarketassistant)
+-   [Kaseem, adolescente con sordera y cegera](/people-use-web/user-stories/#teenager)
```
Comment (espinr) [inline index.es.md]: cegera > ceguera

File: index.es.md [hunk 20]
```diff
@@ -0,0 +1,720 @@
...
+{% include excol.html type="end" %}{% include excol.html type="start" %}
+
+#### Historias de usuario relacionadas con compatibilidad
+{:.no_toc}
+
+{% include excol.html type="middle" %}
+
+-   [Lee, comprador online con cegera para el color](/people-use-web/user-stories/#shopper)
+-   [Alex, reportero con lesión por esfuerzo repetitivo](/people-use-web/user-stories/#reporter)
+-   [Ilya, profesional senior con cegera](/people-use-web/user-stories/#accountant)
```
Comment (espinr) [inline index.es.md]: cegera > ceguera

File: index.es.md [hunk 21]
```diff
@@ -0,0 +1,720 @@
...
+-   [Apartado B. Soporte la producción de contenido accesible](https://www.w3.org/TR/ATAG20/#part_b)
+
+{% include excol.html type="end" %}{% include excol.html type="start" %}
+
+#### Historias de usuario relacionadas con la ayuda en la entrada de datos
+{:.no_toc}
+
+{% include excol.html type="middle" %}
+
+-   [Lee, comprador online con cegera para el color](/people-use-web/user-stories/#shopper)
```
Comment (espinr) [inline index.es.md]: cegera > ceguera

File: index.es.md [hunk 22]
```diff
@@ -0,0 +1,720 @@
...
+
+{% include excol.html type="end" %}{% include excol.html type="start" %}
+
+#### Historias de usuario relacionadas con predecibilidad
+{:.no_toc}
+
+{% include excol.html type="middle" %}
+
+-   [Alex, reportero con lesión por esfuerzo repetitivo](/people-use-web/user-stories/#reporter)
+-   [Ilya, profesional senior con cegera](/people-use-web/user-stories/#accountant)
```
Comment (espinr) [inline index.es.md]: cegera > ceguera

File: index.es.md [hunk 23]
```diff
@@ -0,0 +1,720 @@
...
+-   [Preety, estudiante de secundaria con trastorno por déficit de atención e hiperactividad y dislexia](/people-use-web/user-stories/#classroomstudent)
+-   [Yun, retirada, con baja visión, temblores en manos y leve perdida de memoria a corto plazo](/people-use-web/user-stories/#retiree)
+-   [Luis, asistente en un supermercado con síndrome de Down](/people-use-web/user-stories/#supermarketassistant)
+-   [Kaseem, adolescente con sordera y cegera](/people-use-web/user-stories/#teenager)
```
Comment (espinr) [inline index.es.md]: cegera > ceguera

File: index.es.md [hunk 24]
```diff
@@ -0,0 +1,720 @@
...
+
+{% include excol.html type="end" %}{% include excol.html type="start" %}
+
+#### Historias de usuario relacionadas con la legibilidad
+{:.no_toc}
+
+{% include excol.html type="middle" %}
+
+-   [Martine, estudiante online con problemas de audición](/people-use-web/user-stories/#onlinestudent)
+-   [Ilya, profesional senior con cegera](/people-use-web/user-stories/#accountant)
```
Comment (espinr) [inline index.es.md]: cegera > ceguera

**Review by espinr — CHANGES_REQUESTED** [review]
@rumoroso, many thanks!!! Some minor typos and suggestions. I think it's a good quality translation. Thanks

**Comment by rumoroso** [comment]
Hi Martín (@espinr ), thanks for the review. It was clear this is not your first time you but also that it was for me ;)
Changes applied

**Comment by shawna-slh** [comment]
Hi @cmunch

Please take a quick look at this page to help with the coordination: https://deploy-preview-19--wai-accessibility-principles.netlify.com/fundamentals/accessibility-principles/es

If all looks good, we'll have more translations to publish! :-)

**Comment by espinr** [comment]
> Hi Martín (@espinr ), thanks for the review. It was clear this is not your first time you but also that it was for me ;)

Yep! Sorry, but I started the review early in the morning, (several meetings and interruptions in the middle) and I finished it several hours after. So I worked on a outdated version. My apologies.

**Pull request merged**